lib/Dialect/FIRRTL/Transforms/BlackBoxReader.cpp

```diff
@@ -185,6 +177,9 @@ void BlackBoxReaderPass::runOnOperation() {
   auto bboxAnno =
       builder.getDictionaryAttr({{builder.getStringAttr("class"),
                                   builder.getStringAttr(blackBoxAnnoClass)}});
+  // Track which modules reference which files
+  DenseMap<Operation *, SmallVector<StringAttr>> verbatimExtmoduleToFile;
```

Comment:
Please use MapVector to prevent non-determinism since verbatimExtmoduleToFile is iterated.